apps/demo-app/app/pages/charts.vue

```diff
@@ -1,7 +1,210 @@
-<script lang="ts" setup></script>
+<script lang="ts" setup>
+import { registerOnyxPlugin } from "@sit-onyx/chartjs-plugin";
+import { iconArrowSmallDown, iconArrowSmallRight, iconArrowSmallUp } from "@sit-onyx/icons";
+import type { ChartData } from "chart.js";
+import { Chart, registerables } from "chart.js";
+import { OnyxHeadline } from "sit-onyx";
+import { Bar, Line, Pie } from "vue-chartjs";
+import { useI18n } from "vue-i18n";
+
+Chart.register(...registerables);
+registerOnyxPlugin(Chart);
+
+const { t } = useI18n();
+
+/**
+ * Charts
+ */
+
+const barChartData: Ref<ChartData<"bar">> = computed(() => {
+  return {
+    labels: ["01.01.2024", "01.02.2024", "01.03.2024", "01.04.2024", "01.05.2024"],
+    datasets: [
+      {
+        label: t("charts.productLines.a"),
+        data: [15000, 18000, 16500, 20000, 19500],
+        borderWidth: 2,
+      },
+      {
+        label: t("charts.productLines.b"),
+        data: [12000, 13500, 14000, 11800, 15200],
+        borderWidth: 2,
+      },
+    ],
+  };
+});
+const pieChartData: Ref<ChartData<"pie">> = computed(() => {
+  return {
+    labels: [
+      t("charts.customerSegments.enterprise"),
+      t("charts.customerSegments.mid-sized"),
+      t("charts.customerSegments.small"),
+      t("charts.customerSegments.private"),
+    ],
+    datasets: [{ data: [55, 25, 15, 5] }],
+  };
+});
+const lineChartData: Ref<ChartData<"line">> = computed(() => {
+  return {
+    labels: ["01.01.2025", "01.02.2025", "01.03.2025", "01.04.2025", "01.05.2025"],
+    datasets: [
+      {
+        label: t("charts.metrics.revenue"),
+        fill: true,
+        data: [150, 165, 180, 200, 215],
+      },
+      {
+        label: t("charts.metrics.netProfit"),
+        fill: true,
+        data: [25, 30, 32, 38, 102],
+      },
+    ],
+  };
+});
+
+/**
+ * Select
+ */
+
+const options = computed(() => [
+  {
+    value: "all",
+    label: t("charts.selectOptions.all"),
+  },
+  {
+    value: "products",
+    label: t("charts.selectOptions.products"),
+  },
+  {
+    value: "services",
+    label: t("charts.selectOptions.services"),
+  },
+]);
+const chartOptions = { maintainAspectRatio: false };
+</script>
 
 <template>
-  <OnyxHeadline is="h1">{{ $t("charts") }}</OnyxHeadline>
+  <OnyxHeadline is="h1">{{ t("charts.pageName") }}</OnyxHeadline>
+
+  <OnyxHeadline is="h3" class="headline">{{ t("charts.rangeAndType") }}</OnyxHeadline>
+  <OnyxForm>
```

Comment:
```suggestion
  <OnyxForm @submit.prevent>
```

Prevent submit so no full page reload is triggered when clicking the submit button